packages/styles/schematics/utils/dom/migration-rule.ts

```diff
@@ -24,7 +24,12 @@ const CHEERIO_OPTIONS: cheerio.CheerioOptions = {
 const PRETTIER_OPTIONS: prettier.Options = {
   parser: 'html',
   plugins: [htmlParser],
-  htmlWhitespaceSensitivity: 'strict'
+  printWidth: 1000,
+  bracketSpacing: true,
+  bracketSameLine: true,
+  htmlWhitespaceSensitivity: 'ignore',
```

Comment:
Are you sure about that? I'm not sure we can assume this will not break any interface.

Reply:
Which option property gives you a headache?

Keep in mind the following:
1. Only manipulated elements will be processed with prettier.format.
2. These elements are given to prettier.format as a single-line-string, because they are preprocessed by cheerio.